package.json

```diff
@@ -37,10 +37,10 @@
     "lab": "./bin/lab"
   },
   "scripts": {
-    "test": "./bin/_lab -fL -t 100 -m 3000",
+    "test": "node ./bin/_lab -fL -t 100 -m 3000",
     "posttest": "npm run lint-md",
-    "test-cov-html": "./bin/_lab -fL -r html -m 3000 -o coverage.html",
-    "lint": "./bin/_lab -d -f -L",
+    "test-cov-html": "node ./bin/_lab -fL -r html -m 3000 -o coverage.html",
```

Comment:
Not directly related to this PR, but the previous command didn't work on Windows, because the `#! /bin/node` trick obviously doesn't work on Windows.

Some tests keep failing on my Windows 10 machine, but with this change at least I can run them.